src/file_layer.ts

```diff
@@ -110,4 +141,14 @@ export class FileLayer extends AbstractEmsService {
     }
     return this._config.formats[0];
   }
+
+  private _getFormatOfType(type: EMSFormatTypeStrings): EMSFormats {
+    const requestedFormat = this._config.formats.find((format) => {
+      return format.type === type;
+    });
+    if (!requestedFormat) {
+      return this._getDefaultFormat();
+    }
+    return requestedFormat;
```

Comment:
Would this simplify the return?

```suggestion
    return requestedFormat || this._getDefaultFormat;
```